pkg/container-utils/testutils/options.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutils
+
+const (
+	DefaultContainerName  = "test-container"
+	DefaultContainerImage = "docker.io/library/busybox"
+)
+
+type Option func(*containerOptions)
+
+type containerOptions struct {
+	name           string
+	image          string
+	seccompProfile string
+	wait           bool
+	logs           bool
+	removal        bool
+}
+
+func defaultContainerOptions() *containerOptions {
+	return &containerOptions{
+		name:    DefaultContainerName,
+		image:   DefaultContainerImage,
+		logs:    true,
+		wait:    true,
+		removal: true,
+	}
+}
+
+func WithName(name string) Option {
+	return func(opts *containerOptions) {
+		opts.name = name
+	}
+}
+
+func WithImage(image string) Option {
+	return func(opts *containerOptions) {
+		opts.image = image
+	}
+}
+
+func WithSeccompProfile(profile string) Option {
+	return func(opts *containerOptions) {
+		opts.seccompProfile = profile
+	}
+}
+
+func WithoutWait() Option {
+	return func(opts *containerOptions) {
+		opts.wait = false
+	}
+}
+
+func WithWait() Option {
+	return func(opts *containerOptions) {
+		opts.wait = true
+	}
+}
+
+func WithoutRemoval() Option {
+	return func(opts *containerOptions) {
+		opts.removal = false
+	}
+}
+
+func WithRemoval() Option {
+	return func(opts *containerOptions) {
+		opts.removal = true
+	}
+}
```

Comment:
Do we really need these two functions? Maybe mix them into one or just define the one that sets the opposite to the default value?